src/RouteNotFoundException.php

```diff
@@ -2,11 +2,12 @@
 
 namespace Yiisoft\Router;
 
+use RuntimeException;
 use Throwable;
 
-final class RouteNotFoundException extends \RuntimeException
+final class RouteNotFoundException extends RuntimeException
 {
-    public function __construct($routeName = '', $code = 0, Throwable $previous = null)
+    public function __construct(?string $routeName = '', ?int $code = 0, Throwable $previous = null)
```

Comment:
Why `$routeName` and `$code` is nullable?

Reply:
There is really no reason for it